metrics/grafana/provisioning/dashboards/home.json

```diff
@@ -67,9 +67,7 @@
         "minVizWidth": 75,
         "orientation": "auto",
         "reduceOptions": {
-          "calcs": [
-            "lastNotNull"
-          ],
+          "calcs": ["lastNotNull"],
```

Comment:
A screenshot of the modified/added visualizations in the PR description would be nice.

Reply:
Done